CorsixTH/Lua/hospital.lua

```diff
@@ -78,11 +80,12 @@ function Hospital:Hospital(world, avail_rooms, name)
   self.concurrent_epidemic_limit = level_config.gbv.EpidemicConcurrentLimit or 1
 
   -- Initial values
-  self.interest_rate = interest_rate
+  self.interest_rate = interest_rate / 10000
```

Comment:
This is weird, you have an interest rate, divide by 1000, and it's again an interest rate.
Obviously one of them is a factor 1000 higher than the other, so giving them both the same name is confusing.

Either `interest_rate` should have a different name imho, or you should apply the division at the line above where you copy the configuration value into the `interest_rate`, so you don't have to do that here.